server/api/commands_test.go

```diff
@@ -28,15 +28,15 @@ func TestInvalidLoadCmds(t *testing.T) {
 
 func TestLoadNotReady(t *testing.T) {
 	bw := io.NewBufferWriter()
-	cmd := []string{"1s", "1", "0", "1", "1"}
+	cmd := []string{"1s", "1", "1", "0", "1", "1"}
```

Comment:
I am bit thorn between different parameters for `transactions` and `errors` like you have now; and either transactions or errors like before. 
They way you have it is more flexible, but is also more fiddly - you now have to remember to pass 6 parameters to the `test` command in the right order...

do you think is a particular interesting scenario to mix transactions and errors?

also note that (now and before) you can send many commands concurrently, eg. one with transactions and one with errors - with the caveat that results for those are not saved.

Reply:
another possibility is to add a named flag like `--throttle`, we could have `--with-errors` or something like that to keep the required unnamed option list shorter.

im not against this tho, just trying to figure out what is the simpler user interface

Reply:
The reason I added this handling is that - afaik - it reflects the current agent implementation, where transactions and errors can be sent in the same request.

Sending multiple commands concurrently only works in the interactive shell. The non-interactive part is useful for automation, where I think it makes sense to also provide the possibility of sending all types of events concurrently. I removed the profiles that were defining multiple workloads for one test run, and instead used the config options to define the number of events to send. If this additional parameter is too much overhead, we would need to go back to having some kind of predefined profiles to enable sending error and transactions in concurrent workloads. 

I am in favour of explicitly defining the load than having profiles. If you think the overhead of the additional parameter outweighs the advantage of the flexibility though, we can try to find another solution or go back to having some profiles defined.

Reply:
yes, absolutely agree with that! i meant only for the interactive part, keeping the unified `Compose` function.

so previous behaviour is like:
if numSpans == 0 {numTras

Reply:
this comment should be better added to https://github.com/elastic/hey-apm/pull/51/files#diff-2d44b861c274d4df0c6846fbcba00340R31

imagine i wrote it there :)

Reply:
Ah got it, yes I think we could keep it for the interactive part.

Reply:
I pushed the changes. The configured values will always be `transactions, spans,..`. But you can additionally specify `numErrors x`. Let me know what you think.